ddtrace/contrib/internal/google_adk/__init__.py

```diff
@@ -0,0 +1,46 @@
+"""
+The Google ADK integration instruments the Google ADK Python SDK to trace Agent requests.
+
+All traces submitted from the Google ADK integration are tagged by:
+
+- ``service``, ``env``, ``version``: see the `Unified Service Tagging docs <https://docs.datadoghq.com/getting_started/tagging/unified_service_tagging>`_.
+- model used in the request.
+- provider used in the request.
+
+
+Enabling
+~~~~~~~~
+
+The Google ADK integration is enabled automatically when you use
+:ref:`ddtrace-run<ddtracerun>` or :ref:`import ddtrace.auto<ddtraceauto>`.
+
+Alternatively, use :func:`patch() <ddtrace.patch>` to manually enable the Google ADK integration::
+
+    from ddtrace import config, patch
+
+    patch(google_adk=True)
```

Comment:
```suggestion
```


we shouldn't suggest this approach anymore.

honestly, not a big deal, but figure one less thing to remove in the future.

Reply:
we should continue with the doc suggestions of not suggesting `Pin.onto()` or `patch`

Reply:
removed.